tests/system/test_misc.c

```diff
@@ -338,3 +339,64 @@ int cmd_getuids(void)
 
    return 0;
 }
+
+int cmd_exit_cb(int argc, char **argv)
+{
+   int wstatus;
+   int child_pid;
+   long before_cb;
+   long after_cb;
+   int rc;
+
+   rc = do_sysenter_call3(TILCK_CMD_SYSCALL,
+                     TO_PTR(TILCK_CMD_GET_VAR_LONG),
```

Comment:
Coding style: please align with arguments at the first symbol after '(', for example:
```
foo(longArg1,
    longArg2);
```